pkgs/development/python-modules/credstash/default.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, buildPythonPackage, fetchPypi, cryptography, boto3, pyyaml, docutils, nose }:
+{ stdenv, buildPythonPackage, fetchPypi, cryptography, boto3, pyyaml, docutils, pytest }:
 
 buildPythonPackage rec {
   pname = "credstash";
```

place after version, github doesn't allow me to place correct :(
```suggestion
  pname = "credstash";
  disabled = isPy27;
```